lib/coap/phoenix/conn.ex

```diff
@@ -55,6 +56,10 @@ defmodule CoAP.Phoenix.Conn do
     {:ok, nil, req}
   end
 
+  def read_req_body(state, opts) do
```

Comment:
This gets called here by `read_body`: https://github.com/elixir-plug/plug/blob/master/lib/plug/conn.ex#L1034